### PR TITLE
Add code for hostctl on Apple silicon

### LINKED ISSUE #822: arkade get hostctl fails on m1 mac
with reference to that `hostctl` CLI listed here https://github.com/alexellis/arkade#catalog-of-clis , `arkade get hostctl` fails like this:

```
Downloading: hostctl
2022/12/10 12:30:12 Looking up version for hostctl
2022/12/10 12:30:13 Found: v1.1.3
Downloading: https://github.com/guumaster/hostctl/releases/download/v1.1.3/hostctl_1.1.3_macOS_.tar.gz
Error: incorrect status for downloading tool: 404
```
possibly due to M1 mac. Worked on a x86 mac before. Able to build `hostctl` from the code. So not a blocker, just FYI.

### PR DESCRIPTION
**Pull request by rgee0**
Signed-off-by: Richard Gee <richard@technologee.co.uk>

## Description
Seeks to address #822 

## Motivation and Context
- [x] I have raised an issue to propose this change, which has been given a label of `design/approved` by a maintainer ([required](https://github.com/alexellis/arkade/blob/master/CONTRIBUTING.md))
Fixes #822 

## How Has This Been Tested?
I don't have an M1 to test on, looking for support from @lakamsani to test.  Details on how can be found in [Contributing.md](https://github.com/alexellis/arkade/blob/master/CONTRIBUTING.md#test-other-peoples-prs)

## Are you a GitHub Sponsor yet (Yes/No?)

- [ ] Yes
- [x] No

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Documentation

- [ ] I have updated the list of tools in README.md if (required) with `./arkade get -o markdown`
- [ ] I have updated the list of apps in README.md if (required) with `./arkade install --help`

## Checklist:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I've read the [CONTRIBUTION](https://github.com/alexellis/arkade/blob/master/CONTRIBUTING.md) guide
- [x] I have signed-off my commits with `git commit -s`

- [ ] I have tested this on arm, or have added code to prevent deployment

### CODE REVIEW

File: pkg/get/get_test.go [hunk 1]
```diff
@@ -3695,7 +3706,6 @@ func Test_DownloadSOPS(t *testing.T) {
 			version: version,
 			url:     "https://github.com/mozilla/sops/releases/download/v3.7.2/sops-v3.7.2.darwin.arm64",
 		},
-
```
Comment (rgee0) [inline pkg/get/get_test.go:3698]: This is simply removing an empty line

File: pkg/get/tools.go [hunk 2]
```diff
@@ -2130,7 +2130,7 @@ https://github.com/{{.Owner}}/{{.Repo}}/releases/download/{{.Version}}/{{.Name}}
 
 			{{- if eq .Arch "x86_64" -}}
 			{{ $archStr = "64-bit" }}
-			{{- else if eq .Arch "aarch64" -}}
+			{{- else if or (eq .Arch "aarch64") (eq .Arch "arm64") -}}
```
Comment (rgee0) [inline pkg/get/tools.go:2133]: Follows a pattern used elsewhere, like [line 2214](https://github.com/alexellis/arkade/blob/master/pkg/get/tools.go#L2214)

File: pkg/get/get_test.go [hunk 3]
```diff
@@ -3508,7 +3508,7 @@ func Test_DownloadvCluster(t *testing.T) {
 func Test_DownloadHostcl(t *testing.T) {
 	tools := MakeTools()
 	name := "hostctl"
-	version := "v1.1.1"
+	version := "v1.1.3"
```
Comment (rgee0) [inline pkg/get/get_test.go:3511]: Updated the version as ARM on Mac has seemingly only been supporter after v1.1.1

**Comment by alexellis** [comment]
Thanks @rgee0

**Pull request merged**